index.html

```diff
@@ -423,6 +423,24 @@ <h2>
                       abort these steps. 
                     </p>
                   </li>
+                  <li>
+                    <p>If the <a>current settings object</a>'s's [=relevant global object=]'s [=associated Document=]'s
```

Comment:
What if focus is lost after this step executes?

Reply:
The idea is that we start listening to focus loss for that controller at this exact point in time. And enqueue a task when that happens.
The `If` is indeed misleading. How about `Whenever `, or `The first time `.
Or maybe rephrase to `Listen to ... losing focus. The first time it happens, queue...`

Reply:
Updated to `listen to`.
As of moving the step, I think this is good where it is.
Otherwise, we would potentially set FocusChangeDisabled to true before the user chose the surface (say getDisplayMedia is called, picker is shown, and user decides to unfocus/refocus the page before answering the prompt).